.vib/apache/goss/goss.yaml

```diff
@@ -0,0 +1,18 @@
+http:
+  {{ $securePort := .Vars.containerPorts.https }}
+  {{ range .Vars.hostAliases }}
+  https://{{ .ip }}:{{ $securePort }}:
+    status: 200
+    allow-insecure: true
+  {{ end }}
+file:
+  /etc/hosts:
```

Comment:
This could be a nice test to replicate in the rest of our catalog (where it proceeds). It's true this is only used by `apache-exporter`, which is disabled by default, but is good to test it regardless.